app/src/layouts/kanban/kanban.vue

```diff
@@ -53,7 +53,16 @@
 							<router-link :to="getItemRoute(collection, element.id)" class="item">
 								<div v-if="element.title" class="title">{{ element.title }}</div>
 								<img v-if="element.image" class="image" :src="element.image" />
-								<div v-if="element.text" class="text">{{ element.text }}</div>
+								<render-display
+									v-if="element.text && textFieldConfiguration"
+									:collection="collection"
+									:value="element.text"
+									:type="textFieldConfiguration.type"
+									:field="layoutOptions.textField"
+									:display="textFieldConfiguration.meta?.display"
+									:options="textFieldConfiguration.meta?.options"
+									:interface="textFieldConfiguration.meta?.interface"
+								/>
```

Comment:
As per Rijk's suggestion on #19065, I used the `<render-display>` component in order to format the raw HTML string of WYSIWYG fields.

As far as I can tell, this was the most fitting component to use, but it does not seem to work for all types of text / string fields (e.g. markdown).

The `textFieldConfiguration` is explained below.

Reply:
> As far as I can tell, this was the most fitting component to use, but it does not seem to work for all types of text / string fields (e.g. markdown).

The render-display component will simply render whatever display is configured for that particular field, so depending on the configured display, it should work for all field types 👍🏻